src/components/routes/Booking/BookingOptions/SelectPaymentOption/SelectPaymentOption.container.ts

```diff
@@ -40,7 +40,7 @@ const SelectPaymentOptionContainerTablet = styled(SelectPaymentOptionContainerMo
 
 
     .select-payment-quote-desktop {
-      height: 440px;
+      height: auto;
```

Comment:
nice, removing the fixed height is good. is auto the default? 
would that make a long title work? it'll just make the box height taller?

Reply:
`height: auto` is default for height.  let me take a look tomorrow to make sure it works well on all screens

Reply:
yeah. I thought I saw `height: 100%` being declared in mobile but I read it wrong. REMOVED